client/galaxy/scripts/mvc/form/form-parameters.js

```diff
@@ -152,7 +153,38 @@ define(['utils/utils',
             });
         },
 
-        /** Slider field */
+        
+		_fieldPassword: function(input_def){
+		if (input_def.options && input_def.data){
```

Comment:
Hi Aysam, 
When you talk about options, are you talking about the attributes when you specify the parameter in xml? If yes, then the answer is, it supports the following attributes: name, label, value, type.